packages/millicast-sdk/src/types/index.d.ts

```diff
@@ -1359,6 +1371,42 @@ declare module '@millicast/sdk' {
     sendMetadata(message: String | Object, uuid: String): void;
     webRTCPeer?: PeerConnection;
   }
+
+  /**
+   * DRM encryption parameters
+   */
+  export type EncryptionParameters = {
+    /** 16-byte KeyID, in lowercase hexadecimal without separators */
+    keyId: string;
+
+    /** 16-byte initialization vector, in lowercase hexadecimal without separators */
+    iv: string;
+  }
+
+
+  /**
+   * The configuration for DRM playback
+   */
+  export type DRMOptions = {
+    /** The video element */
+    videoElement: HTMLVideoElement;
+
+    /** The video encryption parameters */
+    videoEncParams: EncryptionParameters;
```

Comment:
nitpick : can we have `videoEncryptionParams` ? 
```suggestion
    videoEncryptionParams: EncryptionParameters;
```